model/src/main/kotlin/config/Resolutions.kt

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2017-2018 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.model.config
+
+import com.fasterxml.jackson.annotation.JsonInclude
+
+/**
+ * Resolutions for issues with a repository.
+ */
+class Resolutions(
+    /**
+     * Resolutions for issues with the analysis or scan of the projects in this repository and their dependencies.
+     */
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+    val errors: List<ErrorResolution> = emptyList()
```

Comment:
Note to myself: I was first wondering why this is called "errors" instead of "resolutions" (because they are more the latter than the former), but currently it reads nicer in the serialized output, and we could add a "warnings" field later, too.